src/ensembl/src/content/app/custom-download/sample-data/attributes.ts

```diff
@@ -0,0 +1,208 @@
+import Attributes from '../types/Attributes';
+
+const attributes: Attributes = {
+  genes: {
+    type: 'section',
+    label: 'Genes',
+    id: 'genes',
+    content: [
+      {
+        type: 'checkbox_grid',
+        label: '',
+        id: 'default',
+        options: [
+          {
+            id: 'name',
+            label: 'Gene name',
+            isChecked: true,
+            order: 1
```

Comment:
What't the `order` field for? If it's to order the options one after another, then their relative positions inside the array should be enough for that.

Reply:
If the options needs to be logically ordered, would it not be better to have this field rather than relying on the relative position in the array?

Reply:
That logical ordering would happen on the backend, presumably? Why wouldn't it sort the options appropriately and put them in the array in the correct order?

Reply:
It would be great if it can but it shouldn't be a big problem to remove it later on?

Reply:
It will. But first we need a BE

Reply:
@msmanoj if I understand your point correctly, you are saying that since we don't have a proper backend, we can use any method available to signify order of options.

To which I would say that since we don't have a proper backend, it's up to us to decide how we encode the information about the order of options; and since an array is already, by definition, an ordered list, we should rely on the order of options in an array and not introduce the order field, which is unnecessary and might confuse backend developers at a later point (if they use our mock data as a starting point to model api response).